pages/edge/tutorials/cdn.mdx

```diff
@@ -0,0 +1,208 @@
+No package found or specified.
+What should the package be called? It will be published under wasmer: <your-app-name> # This will be the package name of your CDN
+What should be the name of the app? <NAME>.wasmer.app: <your-app-name> # This will be the name of your app
+Would you like to publish the app now? no # Select no for now
+Writing app config to '/path/to/your/project/app.yaml'
+To (re)deploy your app, run 'wasmer deploy'
+```
+
+Select the above configuration mentioned in the comments and press enter.
+
+The above command will initialize a blank project with the following structure.
+
+<FileTree>
+  <FileTree.Folder name="your-top-level-dir" defaultOpen>
+    <FileTree.Folder name="public" defaultOpen>
+      <FileTree.File name="index.html" />
+    </FileTree.Folder>
+    <FileTree.File name="app.yaml" />
+    <FileTree.File name="wasmer.toml" />
+  </FileTree.Folder>
+</FileTree>
+
+### Changing configuration for the CDN
+
+As you might notice, this project uses the `static-web-server` package which is a static web server written in Rust.
+
+Most notably, it supports changing the compression level, cache control, cors, and more.
+All these configurations can be super useful when configuring a CDN.
+
+So, let's add a `config.toml` file in your top level directory for `static-web-server`'s configuration.
+
+```toml
+[general]
+
+#### Address & Root dir
+host = "::"
+port = 8080
+root = "/app/public" # 👈🏼 Change the root directory to the public directory for static assets
+```
+
+<Callout type="info" emoji="ℹ️">
+  The above configuration is the default configuration of the
+  `static-web-server` package. More info about it can be found
+  [here](https://static-web-server.net/configuration/config-file/).
+</Callout>
+
+Now, let's add the following configuration to the `wasmer.toml` file.
+
+```yaml
+[package]
+name = "<your-namespace>/<your-app-name>"
+version = "0.1.0"
+description = "<your-namespace>/<your-app-name> CDN on Wasmer Edge"
+
+[dependencies]
+"wasmer/static-web-server" = "^1"
+
+[fs]
+"/app" = "." # 👈🏼 Add the app directory as a volume mapping it to the root directory
+
+[[command]]
+name = "script"
+module = "wasmer/static-web-server:webserver"
+runner = "https://webc.org/runner/wasi"
+
+[command.annotations.wasi] # 👈🏼 Add config.toml as an argument for the webserver
+main-args = ["-w", "/app/config.toml"]
+```
+
+### Adding files for hosting
+
+Now, let's add some files to the `public` directory.
+
+```bash
+$
+$ touch public/index.html
+```
+
+I'll also add an image called `image.jpg` to the `public` directory.
+
+And now our `index.html` file looks like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <img src="./image.jpg" width="600" />
+  </body>
+</html>
+```
+
+You can also add other files like css, js, videos, pdfs, etc.
```

Comment:
I reviewed this further. HTML files are not usually used in CDNs. Are static assets, such as `.jpg`, `.png`, `.pdf`,`.js` and more. Please replace this and use images and `.js` instead Just so people know what urls they got now

Reply:
I changed the tutorial to serve three image files:
```
https://wasmer-edge-as-cdn.wasmer.app/image1.jpg
https://wasmer-edge-as-cdn.wasmer.app/image2.jpg
https://wasmer-edge-as-cdn.wasmer.app/image3.jpg
```